packages/components/src/templates/header.stories.ts

```diff
@@ -57,41 +57,166 @@ export const SampleA = {
               <sd-icon
                 title="Unsere weiteren Auftritte"
                 name="system/website"
-                library="global-resources"
                 class="text-xl absolute -ml-[4px]"
               ></sd-icon>
             </div>
           </sd-navigation-item>
           <sd-navigation-item>
```

Comment:
Here we have a little problem: We're proposing a search icon, but do not make any suggestion what should happen, when clicking on it. Should it open a modal? Should it slide out to an text input? Should it do anything at all?

For the moment, I at least would suggest to make an alert, that at least SOMETHING happens. Or you could make it a link, indicating that we would switch to an external search page.

Reply:
@mariohamann isn't this the case for ALL buttons/items in the header? If think we at least add links to each of the item or (in case of a button) trigger an alert on the page.

Reply:
Hey @karlbaumhauer and @mariohamann,
I believe it's unnecessary to make everything clickable, as this seems to be a template, not a real case example. IMO making every element interactive would add unnecessary complexity to the code, making it harder to clean up and reuse.
For now, I've made only the search icon clickable in each sample. It opens an alert that says, "This could open a search bar.